### ISSUE TITLE
Hate bein' Annoying.

### ISSUE THREAD

**Issue by Catalylia** [stonedDiscord/serverD#23]
Due to the fact it is not even on the change log it may not even be an active command yet and if so I am just being stupid. But /g goes not seem to work.  The format "/g Message" gives no response for both Mod and Admin level, I have not attempted member level. Is there something I need to add to the poker/settings to enable this? Do I try it on a clean and then re-add all the personalized settings folder by folder?

I also notice something that may or may not be serverD's fault. But I can no longer view a log file that is in use. I.E: If the server is using "serverlog67.log" I can't use notepad ++ or anything else to open it. I could open serverlog66 because it's not in use, but I can no longer open a log that serverD is using. I used to do this to review things that just happened while the server was still running. Is this intentional or is this something on my end?

**Comment by stonedDiscord** [comment]
the logging didn't work at all for some people so i tried disabling shared read

/g is not yet built, so far it only exists in source

**Comment by Catalylia** [comment]
Bad as it sounds I hope this is not the thing that solves the issue, since that would seriously nerf a running server's ability to log check on the fly if an issue were occurring. Maybe they have to go into the settings on the program itself and point to the log file? I used to notice that even if I had a proper directory set up in the poker, it wouldn't work unless I went to the server program and did "Config> Log File... > then checked logging after selecting said file. (Then Reloading serv.) I guess it was a windows sharing permission thing like you were saying, but if that fixes their issue then it would be better than nerfing the program itself.
